Add docs for addressable color wipe gradient feature (#3339)
OttoWinter/esphomedocs: components/light/index.rst

  - **blue** (*Optional*, percentage): The percentage the blue color channel should be on. Defaults to ``100%``.
  - **random** (*Optional*, boolean): If set to ``true``, will overwrite the RGB colors by a new, randomly-chosen
    color each time. Defaults to ``false``.

>   - **num_leds** (**Required**, positive int): The number of leds of this type to have before moving on to the next color. If ``gradient`` is true, it's the number of leds to transition to the next color over.

Reviewer: ```suggestion
  - **num_leds** (**Required**, positive int): The number of LEDs of this type to have before transitioning to the next color. If ``gradient`` is true, this will be the number of LEDs over which the color transition will occur.
```
